.dockerignore

```diff
@@ -1,6 +1,5 @@
 # build folders and similar which are not needed for the docker build
 target
-docker
```

Comment:
This is not a good idea, at least you need to put `docker/volumes` and `docker/federation/volumes`

Reply:
I need to be able to COPY the `docker-build.sh` but I can't if this is excluded from the docker build context, I can see though if it's possible to exclude docker folder but add `docker-build.sh` as an exception.